### PR TITLE
Timezones

### PR DESCRIPTION
**Pull request by ethanjurman**
# Timezone BTC-APP
image

Uses moment-timezone to make guess at timezone region

### CODE REVIEW

**Comment by stevekroh** [comment]
What gets returned from moment.tz?

**Comment by ethanjurman** [comment]
moment.tz.guess gives us a best estimate (using available resources) to figure out the current timezone.

File: src/js/store.js [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
 /*esfmt-ignore-start*/
 import notifications from './reducers/notifications';
-import points        from './reducers/points';
-import tracks        from './reducers/tracks';
-import settings      from './reducers/settings';
-import network       from './reducers/network';
-import map           from './reducers/map';
-import filters       from './reducers/filter';
-import account       from './reducers/account';
-import drawer        from './reducers/drawer';
```
Comment (JRJurman) [inline src/js/store.js]: Undo the changes in this file... This is an issue with windows esfmt-ignore not working.

**Comment by JRJurman** [comment]
Looks good to me :+1:
